src/views/notifications/Latest.vue

```diff
@@ -0,0 +1,324 @@
+                                placeholder="Enter Title"
+                                v-model="this.single_notification.title"
+                              />
+                              <div v-if="this.validation_error?.title != null" class="text-theme-6 mt-2 mb-4">
+                                {{ this.validation_error?.title[0] }}
+                              </div>
+                            </div>
+                          </div>
+                          <div class="col-span-12 xxl:col-span-12 mb-4">
+                            <div>
+                              <label for="update-title-notification" class="form-label">
+                                Content
+                              </label>
+                              <textarea
+                                id="update-content-notification"
+                                :class="'form-control' + (this.validation_error?.content != null ? ' border-theme-6' : '')"
+                                placeholder="Enter Content"
+                                rows="2"
+                                v-model="this.single_notification.content"
+                              />
+                              <div v-if="this.validation_error?.content != null" class="text-theme-6 mt-2 mb-4">
+                                {{ this.validation_error?.content[0] }}
+                              </div>
+                            </div>
+                          </div>
+                          <div class="col-span-12 xxl:col-span-6 mb-4">
+                            <div>
+                              <label for="update-color-notification" class="form-label">
+                                Color
+                              </label>
+                              <input
+                                id="update-color-notification"
+                                type="color"
+                                :class="'form-control' + (this.validation_error?.color != null ? ' border-theme-6' : '')"
+                                placeholder="Enter notification reason"
+                                v-model="this.single_notification.color"
+                              />
+                              <div v-if="this.validation_error?.color != null" class="text-theme-6 mt-2 mb-4">
+                                {{ this.validation_error?.color[0] }}
+                              </div>
+                            </div>
+                          </div>
+                          <div class="col-span-12 xxl:col-span-6 mb-4">
+                            <div>
+                              <label class="form-label">
+                                Type
+                              </label>
+                              <TailSelect
+                                v-model="this.single_notification.type"
+                                :class="'form-control' + (this.validation_error?.type != null ? ' border-theme-6' : '')"
```

Comment:
Unnecessary `!= null`

Reply:
Solved in https://github.com/KrbDevelopment/Articly-Frontend/commit/5172bc53bad0121ee21691def6813959202839fa